frontend/src/components/Chat/ChannelList.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-card>
+    <v-container>
+      <v-row no-gutters align="center">
+        <v-col cols="10">{{ title }}</v-col>
+        <v-col cols="1">
+          <channel-join-dialog
+            @channel-join-event="handleChannelJoin"
+            :joinableChannels="getJoinableChannels"
+          ></channel-join-dialog>
+        </v-col>
+        <v-spacer></v-spacer>
+      </v-row>
+      <v-row>
+        <v-divider></v-divider>
+      </v-row>
+      <v-row>
+        <v-col>
+          <v-list dense>
+            <v-list-item-group
+              v-model="selectedChannel"
+              color="primary"
+              @change="handleChannelSelection($event)"
+            >
+              <v-list-item v-for="(item, i) in channels" :key="i">
+                <v-tooltip bottom>
+                  <template v-slot:activator="{ on, attrs }">
+                    <v-list-item-icon v-bind="attrs" v-on="on">
+                      <v-icon
```

Comment:
I couldn't get the unread marker to get displayed. I had the frontend opened in a tab identified as a first user, and the api opened in another identified as a second user, and when I posted a message with the api on a common channel, nothing happened.

Reply:
For the moment, this only works when sending messages through the frontend (on sockets). Messages sent through the API won't be sent immediately because the ChatGateway doesn't know about those. If we want that behavior, then we'll need to add backend support for it (possibly using server events when messages are created so that the Gateway can know to emit to the clients)

But it is possible to test the unread marker by using two separate browser tabs/windows